doc/query.md

```diff
@@ -76,6 +81,54 @@ if (name == null) {
 
 If an index needs to be changed, first delete the existing index by calling `deleteIndexNamed(String indexName)` where the argument is the index name, then call  `ensureIndexed(List<Object> fieldNames, String indexName)` with the new definition.
 
+#### Managing a Text Index
+
+In order to perform text searches on documents in a datastore a text index needs to be created.  Our text indexing process leverages SQLite's full text search (FTS) functionality, therefore SQLite FTS needs to be available for a text index to be created.
```

Comment:
This section has for the most part been re-written so I believe that most of these comments should be answered by the re-write.  Done in a24a9212694f953235bd45f01096d3f3b8a27053.  I'll do some more research on Android versions and support for SQLite FTS and revisit this later.  Perhaps in another PR?